html-aam/customized-select/optgroup-name-desc.tentative.html

```diff
@@ -0,0 +1,265 @@
+        <option>test
+    </optgroup>
+</select>
+
+<!-- 
+test 4: name = bar 
+        description = foo
+     aria-labelledby provided which wins out over legend, but legend is rendered, so still
+     expose it as a description so the information is at least still available to AT 
+-->
+<select>
+    <button><selectedcontent></selectedcontent></button>
+    <optgroup aria-labelledby=b data-testname="optgroup aria-labelledby vs legend" data-expectedlabel="bar" class="ex">
+        <div id=b>bar</div> <!-- invalid per content model -->
+        <legend>foo</legend>
+        <option>test
+    </optgroup>
+</select>
+
+<!--
+test 5: name = foo 
+        description = bar
+-->
+<select>
+    <button><selectedcontent></selectedcontent></button>
+    <optgroup title=bar data-testname="optgroup legend vs title" data-expectedlabel="foo" class="ex">
+        <legend>foo</legend>
+        <option>test
+    </optgroup>
+</select>
+
+
+<!-- title attribute tests
+
+test 1: name = foo 
+        description = n/a
+-->
+<select>
+    <button><selectedcontent></selectedcontent></button>
+    <optgroup title=foo data-testname="optgroup title" data-expectedlabel="foo" class="ex">
+        <option>test
+    </optgroup>
+</select>
+
+<!--
+test 2: name = foo 
+        description = n/a
+        empty label would otherwise result in no accName - use title as fallback -->
+<select>
+    <button><selectedcontent></selectedcontent></button>
+    <optgroup label title=foo data-testname="optgroup title vs empty label attr" data-expectedlabel="foo" class="ex">
+        <option>test
+    </optgroup>
+</select>
+
+<!--
+test 3: name = foo 
+        description = n/a
+        empty legend would otherwise result in no accName - use title as fallback 
+-->
+<select>
+    <button><selectedcontent></selectedcontent></button>
+    <optgroup title=foo data-testname="optgroup title vs empty legend" data-expectedlabel="foo" class="ex">
+        <legend></legend>
+        <option>test
+    </optgroup>
+</select>
+
+<!--
+test 4: name = foo
+        description = n/a
+        empty aria-label would otherwise result in no accName - use title as fallback 
+-->
+<select>
+    <button><selectedcontent></selectedcontent></button>
+    <optgroup aria-label title=foo data-testname="optgroup title vs empty aria-label" data-expectedlabel="foo" class="ex">
+        <legend></legend>
+        <option>test
+    </optgroup>
+</select>
+
+<!--
+test 5: name = foo
+        description = n/a
+        aria-labelledby points to empty element which would otherwise result in no accName - use title as fallback 
+-->
+<select>
+    <button><selectedcontent></selectedcontent></button>
+    <optgroup aria-labelledby=f title=foo data-testname="optgroup title vs aria-labelledby pointing to empty element" data-expectedlabel="foo" class="ex">
+        <div id=f></div>
+        <legend></legend>
+        <option>test
+    </optgroup>
+</select>
+
+<script>
+AriaUtils.verifyGenericRolesBySelector(".ex-generic");
```

Comment:
From the wpt.fyi commit hooks. 😉
> ERROR message: Selector ".ex-generic" should match at least one element.

```suggestion
AriaUtils.verifyGenericRolesBySelector(".ex");
```